inst/csv/OMOP_CDMv5.4_Concept_Level.csv

```diff
@@ -1,2342 +1,527 @@
-MEASUREMENT,MEASUREMENT_CONCEPT_ID,3013531,Gardnerella vaginalis rRNA [Presence] in Genital specimen by Probe,,,,,,,,,,,,,,,,,,,,,,5,Approved UNIT_CONCEPT_IDs for the given MEASUREMENT_CONCEPT_ID
-MEASUREMENT,MEASUREMENT_CONCEPT_ID,3016520,Beta globulin [Mass/volume] in Serum or Plasma by Electrophoresis,,,,,,,,,,,,,,,,,,,,,"8636,8713,8725,8748,8751,8817,8820,8837,8840,8842,8845,8859,8861,8950,9028,9503,9514,9530,9532,9560,9564,9625,32964,32965,44777535,44777592,44777638,45956701",5,Approved UNIT_CONCEPT_IDs for the given MEASUREMENT_CONCEPT_ID
-MEASUREMENT,MEASUREMENT_CONCEPT_ID,3017651,SCL-70 extractable nuclear IgG Ab [Presence] in Serum,,,,,,,,,,,,,,,,,,,,,,5,Approved UNIT_CONCEPT_IDs for the given MEASUREMENT_CONCEPT_ID
-MEASUREMENT,MEASUREMENT_CONCEPT_ID,3017680,Myelocytes [#/volume] in Blood by Manual count,,,,,,,,,,,,,,,,,,,,,"8647,8695,8712,8734,8784,8785,8799,8815,8816,8829,8848,8888,8931,8938,8961,8980,9156,9157,9158,9245,9254,9257,9423,9426,9435,9436,9442,9444,9445,9446,32706,44777520,44777558,44777561,44777562,44777569,44777575,44777580,44777588,45891003",5,Approved UNIT_CONCEPT_IDs for the given MEASUREMENT_CONCEPT_ID
-MEASUREMENT,MEASUREMENT_CONCEPT_ID,3018006,Microscopic observation [Identifier] in Cervix by Cyto stain.thin prep,,,,,,,,,,,,,,,,,,,,,,5,Approved UNIT_CONCEPT_IDs for the given MEASUREMENT_CONCEPT_ID
-MEASUREMENT,MEASUREMENT_CONCEPT_ID,3019210,Glucose [Mass/volume] in Body fluid,,,,,,,,,,,,,,,,,,,,,"8636,8713,8725,8748,8751,8817,8820,8837,8840,8842,8845,8859,8861,8950,9028,9503,9514,9530,9532,9560,9564,9625,32964,32965,44777535,44777592,44777638,45956701",5,Approved UNIT_CONCEPT_IDs for the given MEASUREMENT_CONCEPT_ID
-MEASUREMENT,MEASUREMENT_CONCEPT_ID,3019370,Meperidine [Mass/volume] in Urine by Confirmatory method,,,,,,,,,,,,,,,,,,,,,"8636,8713,8725,8748,8751,8817,8820,8837,8840,8842,8845,8859,8861,8950,9028,9503,9514,9530,9532,9560,9564,9625,32964,32965,44777535,44777592,44777638,45956701",5,Approved UNIT_CONCEPT_IDs for the given MEASUREMENT_CONCEPT_ID
-MEASUREMENT,MEASUREMENT_CONCEPT_ID,3023548,Base deficit in Arterial blood,,,,,,,,,,,,,,,,,,,,,"8729,8736,8745,8749,8753,8839,8843,8875,9440,9490,9491,9501,9553,9557,9559,9575,9586,9587,9588,9591,9608,9621,9631,9632,9654,9673,45891014",5,Approved UNIT_CONCEPT_IDs for the given MEASUREMENT_CONCEPT_ID
-MEASUREMENT,MEASUREMENT_CONCEPT_ID,3024653,FEV1,,,,,,,,,,,,,,,,,,,,,"8519,8583,8587,8686,9261,9263,9271,9277,9283,9285,9286,9287,9288,9292,9293,9296,9300,9301,9303,9304,9314,9316,9317,9318,9366,9367,9382,9383,9390,9391,9393,9394,9412,9416,9482,9486,9515,9520,9535,9606,9628,9643,9665,44777531,44777662",5,Approved UNIT_CONCEPT_IDs for the given MEASUREMENT_CONCEPT_ID
-MEASUREMENT,MEASUREMENT_CONCEPT_ID,3025313,Albumin [Mass/volume] in Body fluid,,,,,,,,,,,,,,,,,,,,,"8636,8713,8725,8748,8751,8817,8820,8837,8840,8842,8845,8859,8861,8950,9028,9503,9514,9530,9532,9560,9564,9625,32964,32965,44777535,44777592,44777638,45956701",5,A
```

Comment:
Is there a reason for not including 4041262 - Procedure on male genital system?

Reply:
The OMOP vocabulary hierarchy is wrong in some concepts here, HCPCS "procedures on prostate OR perineum" has both parents "procedures on prostate" and "procedures on perineum". but it's OR and these codes are used in females.
Once the vocabulary is fixed, we can add these

Reply:
I was not able to find the concepts in the example you are referring to.

In any case, I suggest to not wait for a vocabulary update, and already add a set of more specific concepts for male procedures that do not include prostate. e.g. these three:
 - 4040708 - Procedure on scrotum
 - 4043199 - Procedure on penis
 - 4040577 - Procedure on testis

Reply:
yes, sorry, it was Procedure on scrotum that lead to concept:
2313992 | 93975 | Duplex scan of arterial inflow and venous outflow of abdominal, pelvic, scrotal contents and/or retroperitoneal organs; complete study
Procedure on prostate leads to the 2211770	76872	Ultrasound, transrectal, which can be done for females as well.
So, I ended up with the following list
4250917 - Operation on prostate
4077750 - Operation on scrotum
4043199 - Procedure on penis
4040577 - Procedure on testis